tests/ui/iterators/iter-macro-not-async-closure.rs

```diff
@@ -7,9 +7,7 @@
 // explanation is moved to the `help` instead of the span label.
 //
 //@ edition: 2024
-//@ revisions: narrow wide
-//@[narrow] compile-flags: --diagnostic-width=20
-//@[wide] compile-flags: --diagnostic-width=300
```

Comment:
This looks almost like the test intentionally wants to cover different widths?

Reply:
(Yes, I'm just checking if the *original* version of the test passes without manually specifying diag width, since it failed in CI but not locally)